openshift_xinetd_example.conf

```diff
@@ -0,0 +1,49 @@
+# https://linux.die.net/man/5/xinetd.conf
+
+service openshift-api
+{
+    flags           = IPv4
+    bind            = <Lab_Host_IP>
+    disable         = no
+    type            = UNLISTED
+    socket_type     = stream
+    protocol        = tcp
+    user            = root
+    wait            = no
+    redirect        = <IPv6_API_Address> 6443
+    port            = 6443
+    only_from       = 0.0.0.0/0
+    per_source      = UNLIMITED
+}
+
+service openshift-ingress-ssl
+{
+    flags           = IPv4
+    bind            = <Lab_Host_IP>
+    disable         = no
+    type            = UNLISTED
+    socket_type     = stream
+    protocol        = tcp
+    user            = root
+    wait            = no
+    redirect        = <IPv6_Ingress_Address> 443
+    port            = 443
+    only_from       = 0.0.0.0/0
+    per_source      = UNLIMITED
+}
+
+service openshift-ingress
+{
+    flags           = IPv4
```

Comment:
Does this example only work for ipv4?  The default is ipv6 so we probably need the default example config to either work with ipv6, or ideally either protocol?

Reply:
It works with IPv6.
While I haven't got a canonical answer yet, it seems like the IPv4 flag is for the bind directive, which is there to indicate an address or an interface that the service is bound/exposed on.

Reply:
Confirmed that the flag only affects the `bind` directive. So if at some point we will start addressing our hosts via IPv6, we can change the flag as well :)